beacon-chain/blockchain/metrics.go

```diff
@@ -130,6 +130,14 @@ var (
 		Name: "sync_head_state_hit",
 		Help: "The number of sync head state requests that are present in the cache.",
 	})
+	stateBalanceCacheHit = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "balance_cache_hit",
```

Comment:
there names are too similar to the other metrics like `total_effective_balance_cache_hit`

Reply:
why dont define these metrics closer to its usages inside `state_balance_cache.go`?

Reply:
> why dont define these metrics closer to its usages
I'm following the conventions of the blockchain package here, all the prom metric values are defined in `metrics.go`.
> there names are too similar to the other metrics
how about `state_balance_cache_(hit|miss)`?

Reply:
for some reason i thought the cache resides in `cache` pkg, it makes sense that the metric is here, i dont want to go into another debate where the cache should live ha!

`state_balance_cache` is fine, `justified_state_balance_cache` is also fine